lib/Test/Nginx/Util.pm

```diff
@@ -2364,6 +2373,23 @@ retry:
             #warn "pid file not found";
         }
     }
+
+shutdown_error_log:
+    if (defined $block->shutdown_error_log) {
```

Comment:
The location of this piece of code looks wrong to me since for normal testing mode, the nginx is not even shut down yet at this point. Have you actually tested this in normal testing mode? I could be wrong though.